gapic/templates/testing/constraints-3.13.txt.j2

```diff
@@ -1,4 +1,19 @@
-# -*- coding: utf-8 -*-
-{% block constraints %}
-{% include "testing/_default_constraints.j2" %}
-{% endblock %}
+{% from '_pypi_packages.j2' import pypi_packages %}
+# This constraints file is used to check that the latest
+# major versions of dependencies are supported in setup.py.
+# List all library dependencies and extras in this file.
+# Require the latest major version be installed for each dependency.
+# e.g., if setup.py has "google-cloud-foo >= 1.14.0, < 2.0.0",
+# Then this file should have google-cloud-foo>=1
+google-api-core>=2
+google-auth>=2
+proto-plus>=1
+protobuf>=6
+{% for package_tuple, package_info in pypi_packages.items() %}
+{# Quick check to make sure the package is different from this setup.py #}
```

Comment:
Could you clarify this comment? I think you're checking we don't circularly include this package in its own constraints file. Is that right?

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/2362/commits/b786cc26b9e42d3cf47decc55576aef638ac7b8b